project-proposals/stage-2/stage-2-questionnaire.md

```diff
@@ -0,0 +1,80 @@
+# Stage 2: Incubation
+
+*As a project intending to move to stage 2 of the OpenRail incubation process, please fill out this questionnaire and send it to the Technical Committee. See the [incubation process](../../incubation-process.md) for details about criteria and requirements for new projects and how it works to get new projects into the association.*
+
+*Copy this template and fill in your answers to the questions in the sections below.*
+
+*The document has two sections. In the first section you provide references and evidence that show that you fullfil the criteria for incubation stage 2. These will mostly be links to documentation and other sources of information which show that you fullfil the criteria. Feel free to add any additional information which helps to explain or interprete this information. The second section is a checklist the Technical Committee fills out when it evaluates the proposal.*
```

Comment:
```suggestion
*The document has two sections. In the first section you provide references and evidence that show that you fulfill the criteria for incubation stage 2. These will mostly be links to documentation and other sources of information which show that you fulfill the criteria. Feel free to add any additional information which helps to explain or interpret this information. The second section is a checklist the Technical Committee fills out when it evaluates the proposal.*
```
see
https://www.merriam-webster.com/dictionary/fulfill
and
https://www.merriam-webster.com/dictionary/interpret